Content.Server/GameObjects/Components/Doors/ServerDoorComponent.cs

```diff
@@ -96,6 +98,12 @@ void ICollideBehavior.CollideWith(IEntity entity)
             }
             if (entity.HasComponent(typeof(SpeciesComponent)))
             {
+                if (!entity.TryGetComponent<IMoverComponent>(out var mover)) return;
+                if (!entity.TryGetComponent<ITransformComponent>(out var entityTransform)) return;
+                if (!Owner.TryGetComponent<ITransformComponent>(out var ownerTransform)) return;
```

Comment:
This shouldn't be necessary. As an entity, `Owner` is guaranteed to have a transform component, accessible through `Owner.Transform`. Same for `entity.Transform`.

Reply:
@ComicIronic thanks a lot, I'll keep that in mind for the future. I think there is no point fixing it here, as PJB says this should be fixed in physics.